src/TimeService.Broker/Consumers/CreateWorkTimeConsumer.cs

```diff
@@ -0,0 +1,48 @@
+﻿using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Requests.Time;
+using LT.DigitalOffice.TimeService.Data.Interfaces;
+using LT.DigitalOffice.TimeService.Models.Db;
+using MassTransit;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.TimeService.Broker.Consumers
+{
+    public class CreateWorkTimeConsumer : IConsumer<ICreateWorkTimeRequest>
+    {
+        private readonly IWorkTimeRepository _workTimeRepository;
+
+        private bool CreateWorkTime(ICreateWorkTimeRequest request)
```

Comment:
What for we return bool here?

Reply:
for project service

Reply:
just function is always returns true. there is no return false or throwing errors

Reply:
exc can be thrown if we have problems with db connection